src/metrics/encoding/msgpack/wire_format.md

```diff
@@ -1,115 +0,0 @@
-## Wire format for unaggregated metrics
```

Comment:
Do we have something like this for protobuf?

Reply:
The .proto file should be pretty self explanatory, we had this for msgpack because the code itself was really hard to read lol